### PR TITLE
Add bandwidth plugin support

### PR DESCRIPTION
**Pull request by CallMeFoxie**
Hi, this code adds support for (currently in testing) CNI module for setting bandwidth.

### CODE REVIEW

File: pkg/ocicni/types.go [hunk 1]
```diff
@@ -32,6 +32,16 @@ type NetworkConfig struct {
 	IP string
 }
 
+type BandwidthConfig struct {
+	// IngressRate is a limit for incoming traffic
```
Comment (dcbw) [inline pkg/ocicni/types.go]: Can you specify the units that these are in? eg I believe it's bytes-per-second per CNI conventions. Also, can we do these as uint64s?

Comment (CallMeFoxie) [reply pkg/ocicni/types.go]: done :)

**Comment by CallMeFoxie** [comment]
I ran gofmt on all the PRs.. except this one :) thanks for pointing it out! (that's what I get for using vim)

rebased & force pushed.

**Comment by CallMeFoxie** [comment]
Hmm has the /approve label not been added? :) I can see it in the comment above but not applied :)

**Comment by haircommander** [comment]
@dcbw the /approve didn't go through, can you resend it?

**Pull request merged**